spec/bolt/outputter/human_spec.rb

```diff
@@ -91,6 +91,7 @@
     TABLE
   end
 
+  # TODO: Update when --nodes is deprecated
```

Comment:
I think we should update the outputter now with 
```
USAGE:
bolt task run --targets <target-name> 
```

Reply:
Updated!